reverseString.js

```diff
@@ -0,0 +1,12 @@
+const reverseString = str => {
+    let a = ""
+    let b = ""
+    for(let i = str.length-1; i >= 0; i--){
+        a = str[i].toLowerCase();
+        if (a >="a" && a<= "z"){
+            b += a
+        }
+      }
+      return b;
+    }
```

Comment:
I don't understand the logic behind this, on our next call you will explain the logic here to me and how come it works for other strings. Pretty cool solution btw.

I tested this function with the following and it worked.
```
console.log(reverseString("Just' come-here a!nd-try/it"))
console.log(reverseString("I!m a  li:tt-le t?ea+p)ot"))
```

Reply:
no problem, this was the way my brain figured it out and i was happy when it worked. i'd be happy to explain my thought process. But fair, i suck at explaining things

Reply:
No problem, this was the way my brain figured it out and i was happy when it worked. I'd be happy to explain my thought process. But fair warning, I suck at explaining things